environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/request/EnvironmentRealTimeCostRequest.java

```diff
@@ -4,18 +4,19 @@
 import java.util.List;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.google.api.client.util.Lists;
 
 import io.swagger.annotations.ApiModel;
 
 @ApiModel(value = "EnvironmentRealTimeCostV1Request")
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class EnvironmentRealTimeCostRequest implements Serializable {
 
-    private List<String> environmentCrns;
+    private List<String> environmentCrns = Lists.newArrayList();
```

Comment:
maybe `List.of()` would be sufficient if there is no addition to these list. just a bit of performance gain :)

Reply:
will do in the followup PR